dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
Sounds good--we can always move them into a folder if things start to get cluttered.